tribe-ext-view-tickets/index.php

```diff
@@ -75,10 +128,11 @@ public function show_ticket_page() {
 
 		// Include the class and functions that power this.
 		require_once( 'src/functions/general.php' );
-		require_once( 'src/Tribe/Extension/Tickets_Order_Helper.php' );
+		require_once( 'src/Tribe/Extension/Tickets_Order_Helper2.php' );
```

Comment:
Unfortunately I am using the order helped in another extension as well. If they're running them both side by side, and the old one gets loaded, this extension will break. While this classis backwards compatible, there is no way for us to force the new one to load.

This whole class should disappear in 4.5, when we get a series of builtin ET functions to replace it. Until then, it's basically a workable hack.